packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -250,3 +271,18 @@ class TooltipTheme extends InheritedTheme {
   @override
   bool updateShouldNotify(TooltipTheme oldWidget) => data != oldWidget.data;
 }
+
+/// The mode that will decide when the tooltip will trigger
+enum TooltipTriggerMode {
+  /// Tooltip will not be shown automatically, but can be done manually
+  /// by calling `ensureTooltipVisible`
+  Manual,
+
+  /// Tooltip will be shown automatically after a long press.
+  /// See `GestureDetector.onLongPress` for more details.
+  LongHold,
```

Comment:
I think LongHold may be confusing. Since the gesture is a longPress, it should probably be named the same.